src/plugins/gui/fiatPluginTypes.ts

```diff
@@ -59,6 +60,7 @@ export interface FiatPluginFactoryArgs {
   // io: {
   //   log: EdgeLog, // scoped logs
   // }
+  disablePlugins: DisablePluginMap
```

Comment:
Per convo with @DunningKrugR , looks like we need more granular ability to disable provider plugins and not reuse the buy/sell plugin list.

I would recommend a structure like the following

```
{
    exchangeInfo: {
        buy:  {
            disablePlugins:  {
                moonpay: true, // This refers to a high level plugin.id
                wyre: true,
                creditcard: { // Applies to the creditcard FiatPlugin
                    moonpay: true // Within the creditcard FiatPlugin, this applies to the provider id
                }
            },
        },
        sell: {
            disablePlugins:  {
                simplex: true
                iach: {
                    moonpay: true
                }
            },
        }
    }
}
```

Reply:
done